.gitignore

```diff
@@ -9,3 +9,4 @@ pkg/*
 bin/*
 files/**/cache/
 vendor/cookbooks
+*.swp
```

Comment:
Files like this are specific to your personal editor preferences and would better be managed in a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).